R/reportEmployment.R

```diff
@@ -1,6 +1,5 @@
 #' Computes the employment values (jobs) across different sectors
 #' @param gdx A gdx file output from a REMIND run
```

Comment:
There's a warning in the [GitHub action check](https://github.com/pik-piam/remind2/pull/104/checks) saying that `gdx` is not documented, but here it is. Did you run `lucode2::buildLibrary()`? That should among other things build the documentation

Reply:
works after new commits